src/test/java/com/checkout/instruments/four/CardTokenInstrumentsTestIT.java

```diff
@@ -201,4 +204,14 @@ protected CreateInstrumentTokenResponse createTokenInstrument(final String custo
 
     }
 
+    @Test
+    void shouldFailGetBankAccountFieldFormattingWhenNoOAuthIsProvided() {
+        try {
+            fourApi.instrumentsClient().getBankAccountFieldFormatting(CountryCode.GB, Currency.GBP, BankAccountFieldQuery.builder().build());
+        } catch (final Exception e) {
+            assertTrue(e instanceof CheckoutAuthorizationException);
+            assertEquals("Operation does not support OAUTH authorization type", e.getMessage());
```

Comment:
What do you think about changing this error message to something less confusing, like: "Operation requires <type> authorization type". I remember thinking about this before but I cannot remember why we left it like this.

Reply:
I review this, but I think is more confusing to know which operation requires, with the current message you know that the operation that you're trying to do is not supported, with the new message, we need to provide which `SdkAuthorizationType` are available to use.